src/resolve/resolved-dns-dnssec.c

```diff
@@ -1169,6 +1169,7 @@ int dnssec_verify_rrset_search(
                 DnsResourceRecord **ret_rrsig) {
 
         bool found_rrsig = false, found_invalid = false, found_expired_rrsig = false, found_unsupported_algorithm = false;
+        size_t nvalidations = 0;
```

Comment:
Hmm, `size_t` is what we usually use for counting/sizing objects in memory. This isn't really that.

Reply:
maybe use just `unsigned`?